qdlt/dlt_common.c

```diff
@@ -1946,6 +2188,19 @@ int dlt_file_free(DltFile *file,int verbose)
     return dlt_message_free(&(file->msg),verbose);
 }
 
+int dlt_file_check_version(DltFile *file,int verbose)
+{   
+    PRINT_FUNCTION_VERBOSE(verbose);
+    char first_char =fgetc(file->handle);
+    if (first_char != EOF) {
+        rewind(file->handle);
+    } else {
+        printf("File is empty\n");
```

Comment:
it is bad that low level parsing function directly prints something to the console